documentation/testnet/getting_started/03_deploy_and_execute_demo.md

```diff
@@ -10,162 +10,133 @@ Bugs, usability suggestions, and feedback in general would be greatly appreciate
 
 ## Overview
 
-Learn how to deploy and execute a basic "hello world!" program on Aleo's network using Leo and snarkOS. 
+Learn how to deploy and execute a basic "hello world!" program on Aleo's network using Leo and snarkOS.
 
 ## Usage guide
 
 ### 1. Prerequisites
 
 Make sure you have both Leo and snarkOS installed on your machine.
 
-* To verify if you have Leo, open your terminal and type `leo`. If you don't see something like `zsh: command not found: leo`, you're good
+- To verify if you have Leo, open your terminal and type `leo`. If you don't see something like `zsh: command not found: leo`, you're good
```

Comment:
Is there a rationale behind adding the dashes `-` for bullet points?

Reply:
It's my auto-formatter, let me change it to support `*` over `-` to be in line with the rest of our docs.